src/main/java/homework_6/map_problems_generator/MapProblemsGenerator.java

```diff
@@ -0,0 +1,34 @@
+package homework_6.map_problems_generator;
+
+import java.util.HashMap;
+
+public class MapProblemsGenerator {
+
+    public void makeCollision() {
+
+        HashMap<MapProblemsCollisionGenerator, String> mapForCollisions = new HashMap<>();
+
+        MapProblemsCollisionGenerator obj1 = new MapProblemsCollisionGenerator(10);
+        MapProblemsCollisionGenerator obj2 = new MapProblemsCollisionGenerator(15);
+
+        mapForCollisions.put(obj1, "field");
+        mapForCollisions.put(obj2, "anotherField");
+        System.out.println("mapForCollisions = " + mapForCollisions);
+        System.out.println(obj1.hashCode() == obj2.hashCode());
+    }
+
+    public void makeUnreachedObject() {
+
+        HashMap<MapProblemsMutableGenerator, String> mapForUnreachableObjects = new HashMap<>();
+
+        MapProblemsMutableGenerator obj1 = new MapProblemsMutableGenerator(125);
+        MapProblemsMutableGenerator obj2 = new MapProblemsMutableGenerator(126);
+
+        mapForUnreachableObjects.put(obj1, "field");
```

Comment:
Doesn't seem to work 
![image](https://user-images.githubusercontent.com/58872509/134303172-debde803-b0ca-450f-a07c-5e7990c486f5.png)